core/controllers/reader.py

```diff
@@ -133,7 +157,7 @@ def classify_hard_rule(state, params, input_type, normalized_answer, fs):
     return None
 
 
-def classify_soft_rule(state, params, input_type, normalized_answer, fs):
+def classify_soft_rule(state, params, input_type, normalized_answer):
```

Comment:
A note to developers here: classify_soft_rules() is going to completely change as part of the answer migration (it will no longer call any rule evaluations).